### DISCUSSION TITLE
Python versions

### DISCUSSION THREAD

**Discussion by jhaack** [VOLTTRON/volttron#2812] [General]
Keeping VOLTTRON current with the latest version of Python is a community goal. Could you let us know what OS and Python versions everyone is using?

**Comment by acedrew** [comment]
All of our edge deployments have been moved to Ubuntu 20.04 and Python 3.8+
We will want to migrate to 22.04 ASAP as it's available. Not sure what version of Python will ship, but assume at least 3.9
